app/controllers/staff_controller.rb

```diff
@@ -0,0 +1,7 @@
+class StaffController < StaffAreaController
+
+  def index
+    @users = User.all
```

Comment:
This should be limited to the last 10 users otherwise this will be a HUGE problem in production where there are 100s of users